### ISSUE TITLE
On keepalived failover we saw static routes not applied

### ISSUE THREAD

**Issue by remibergsma** [MissionCriticalCloud/cosmic#778]
Logs show:
```
Oct 31 10:09:44 r-68201-nl2 Keepalived_vrrp[1644]: Netlink: error: Network is unreachable, type=(24), seq=1540825043, pid=0
``` 
which implies the interface was not yet up. Needs investigation.

**Comment by remibergsma** [comment]
Work around: `systemctl reload keepalived`

**Comment by remibergsma** [comment]
Fixed in #792
